### PR TITLE
Refactor the Root README

### PR DESCRIPTION
**Pull request by super3**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,9 +25,33 @@ Use the linked command line interface:
 storj --help
 ```
 
+Installation and Guides
+-------------------------
+  - [Install Guide for Windows, Mac, and Linux](INSTALL.md)
```
Comment (bookchin) [inline README.md]: Need to use complete URL so the docs website links properly.

File: README.md [hunk 2]
```diff
@@ -131,9 +25,33 @@ Use the linked command line interface:
...
+  - [Example 2 - Generating a KeyPair](/example/2-generate-keypair.js)
+  - [Example 3 - Authenticating with a KeyPair](/example/3-authenticate-with-keypair.js)
+  - [Example 4 - Listing Keys](/example/4a-list-keys.js)
+  - [Example 4b - Add/Remove Keys](/example/4b-add-remove-keys.js)
+  - [Example 5a - List Buckets](/example/5a-list-buckets.js)
+  - [Example 5b - Add/Remove Bucket](/example/5b-add-remove-bucket.js)
+  - [Example 6a - Upload File](/example/6a-upload-file.js)
+  - [Example 6b - Download File](/example/6b-download-file.js)
+  - [Example 6c - List Bucket Files](/example/6c-list-bucket-files.js)
+  - [Example 6d - Delete File from Bucket](/example/6d-delete-file-from-bucket.js)
```
Comment (bookchin) [inline README.md]: Same for all of these, use the full URL so the docs website links properly.

File: README.md [hunk 3]
```diff
@@ -131,9 +25,33 @@ Use the linked command line interface:
...
+
+
+
 License
 -------
 
+```
```
Comment (bookchin) [inline README.md]: Why is this now a code block? I removed this earlier. :|

**Comment by bookchin** [comment]
This has been included in #319 with conflicts resolved

**Comment by super3** [comment]
@bookchin Thanks.

**Pull request closed without merging**